routers/user/notification.go

```diff
@@ -79,3 +83,32 @@ func Notifications(c *context.Context) {
 	c.Data["Page"] = paginater.New(int(total), perPage, page, 5)
 	c.HTML(200, tplNotification)
 }
+
+// NotificationStatusPost is a route for changing the status of a notification
+func NotificationStatusPost(c *context.Context) {
+	var (
+		notificationID, _ = strconv.ParseInt(c.Req.PostFormValue("notification_id"), 10, 64)
+		statusStr         = c.Req.PostFormValue("status")
+		status            models.NotificationStatus
+	)
+
+	switch statusStr {
+	case "read":
+		status = models.NotificationStatusRead
+	case "unread":
+		status = models.NotificationStatusUnread
+	case "pinned":
+		status = models.NotificationStatusPinned
+	default:
+		c.Handle(500, "InvalidNotificationStatus", errors.New("Invalid notification status"))
```

Comment:
Minor, but we should include the invalid status (i.e. `statusStr`) in the error message